nomad/structs/structs.go

```diff
@@ -708,6 +708,9 @@ type Node struct {
 	// Datacenter for this node
 	Datacenter string
 
+	// Region for this node
```

Comment:
Hey lets actually remove this. It is better to keep this normalized as we will not have a field string stored in memory for every node that is know by virtue of the region you are talking to.